jennite-vcf-store/src/main/dist/deb/debian/postinst

```diff
@@ -14,10 +14,19 @@ case "$1" in
         . /etc/default/opal
         mkdir -p $OPAL_HOME/plugins
         if [ -d "$OPAL_HOME"/plugins ]; then
+        	OLD_PLUGIN=$(ls -t "$OPAL_HOME"/plugins/ | grep jennite-vcf-store | head -1)
+
           cp -r /usr/share/jennite* $OPAL_HOME/plugins
+
+          if [ ! -z "$OLD_PLUGIN" ]; then
+          	echo "Copying $OLD_PLUGIN/site.properties to new installation."
+						NEW_PLUGIN=$(ls -t $OPAL_HOME/plugins/ | grep jennite-vcf-store | head -1)
+						cp $OPAL_HOME/plugins/$OLD_PLUGIN/site.properties $OPAL_HOME/plugins/$NEW_PLUGIN/
```

Comment:
You could backup the original `site.properties` to `site.properties.pkg` before copying the old one.